src/main/java/com/redislabs/redisgraph/graph_entities/Property.java

```diff
@@ -54,18 +56,33 @@ public void setName(String name) {
     /**
      * @return property type
      */
+    @Deprecated
     public ResultSet.ResultSetScalarTypes getType() {
+        if (type.ordinal() >= 6 && type.ordinal() < 12){
+            return ResultSet.ResultSetScalarTypes.values()[type.ordinal()-6];
```

Comment:
Shouldn't this be like `setType`, adding 6 to the deprecated types rather than the other way around?

Reply:
in setType, you add 6 to the deprecated types and get the valid type
in getType, which is a deprecated call, the caller expects deprecated type.